src/core/server/graph/schema/schema.graphql

```diff
@@ -3218,6 +3218,12 @@ type Comment @cacheControl(maxAge: 5) {
     after: Cursor
   ): CommentsConnection!
 
+  flattenedReplies(
+    first: Int = 10
+    orderBy: COMMENT_SORT = CREATED_AT_ASC
+    after: Cursor
+  ): CommentsConnection!
```

Comment:
Suggestion to instead add a `flattened: Boolean = false` on the `replies` edge instead.